WORKSPACE

```diff
@@ -439,7 +439,6 @@ haskell_register_ghc_nixpkgs(
         "-optc-mmacosx-version-min=10.14",
         "-opta-mmacosx-version-min=10.14",
         "-optl-mmacosx-version-min=10.14",
-        "-optP-mmacosx-version-min=10.14",
```

Comment:
This ends up with an unknown flag. Not sure if we end up calling cpp directly now or if ghc is cpp but either way, there is no point in passing this to cpp anyway so this seems very reasonable.

Reply:
Yes, since 8.8 GHC passes `-optP` flags to gcc via `-Xpreprocessor`, i.e. they go directly to `cpp`. https://github.com/tweag/rules_haskell/pull/1331 includes rules_haskell's workaround and links to a bunch of related issues.